aspnetcore/security/authorization/secure-data.md

```diff
@@ -110,15 +104,34 @@ Append [AddRoles](/dotnet/api/microsoft.aspnetcore.identity.identitybuilder.addr
 
 [!code-csharp[](secure-data/samples/final3/Startup.cs?name=snippet2&highlight=9)]
 
+<a name="rau"></a>
+
 ### Require authenticated users
 
-Set the default authentication policy to require users to be authenticated:
+Set the fallback authentication policy to require users to be authenticated:
 
-[!code-csharp[](secure-data/samples/final3/Startup.cs?name=snippet&highlight=15-99)] 
+[!code-csharp[](secure-data/samples/final3/Startup.cs?name=snippet&highlight=13-99)]
 
- You can opt out of authentication at the Razor Page, controller, or action method level with the `[AllowAnonymous]` attribute. Setting the default authentication policy to require users to be authenticated protects newly added Razor Pages and controllers. Having authentication required by default is more secure than relying on new controllers and Razor Pages to include the `[Authorize]` attribute.
+The preceding highlighted code sets the [fallback authentication policy](xref:Microsoft.AspNetCore.Authorization.AuthorizationOptions.FallbackPolicy). The fallback authentication policy requires ***all*** users to be authenticated, except for Razor Pages, controllers, or action methods with an authentication attribute. For example, Razor Pages, controllers, or action methods with `[AllowAnonymous]` or `[Authorize(PolicyName="MyPolicy")]` use the applied authentication attribute rather than the fallback authentication policy.
+
+The fallback authentication policy:
+
+* Is applied when no authentication attribute is applied.
+* Is part of the [endpoint routing system](xref:fundamentals/routing).
```

Comment:
Is it? It applies to requests with or without endpoint routing.

Reply:
Changed to 
Is applied to endpoint routing when no authentication attribute is applied.

Reply:
This should say something along the lines of:

> Is applied to all requests that do not explicitly specify an authentication policy. For requests served by endpoint routing, this would include any endpoint that does not specify an authorization (❓ because the previous doc says authentication) attribute. For requests served by other middlewares, such as static files, this would apply the policy to all requests.

Reply:
other middleware after the authorization middleware...

Reply:
> For requests served by other middleware after the authorization middleware, such as [static files

Static file middleware comes before auth middleware. Only session and ResponseCaching come after.

Reply:
That ordering is up to the user. They can put StaticFiles after Authorization if they want to require the static files to be authorized. That's actually the easiest way to do static file auth.